Helm 3: style conformance broken

The style conformance is broken which means tests will not run in a dev environment.

```console
$ make test-style
/home/usr1/go/bin/golangci-lint run
pkg/action/upgrade.go:43: File is not `goimports`-ed (goimports)
	Install   bool
	Devel     bool
	Namespace string
	Timeout   time.Duration
	Wait      bool
cmd/helm/upgrade_test.go:239: File is not `gofmt`-ed with `-s` (gofmt)
		Templates: []*chart.File{&chart.File{Name: "templates/configmap.yaml", Data: configmapData}},
Makefile:81: recipe for target 'test-style' failed
make: *** [test-style] Error 1

$ make test
go build  -tags '' -ldflags '-w -s -X helm.sh/helm/internal/version.gitCommit=6b96d2158b0db94f1e1792df2a0e5e7dfa6674b0 -X helm.sh/helm/internal/version.gitTreeState=clean' -o /home/usr1/go/src/helm.sh/helm/bin/helm helm.sh/helm/cmd/helm
/home/usr1/go/bin/golangci-lint run
pkg/action/upgrade.go:43: File is not `goimports`-ed (goimports)
	Install   bool
	Devel     bool
	Namespace string
	Timeout   time.Duration
	Wait      bool
cmd/helm/upgrade_test.go:239: File is not `gofmt`-ed with `-s` (gofmt)
		Templates: []*chart.File{&chart.File{Name: "templates/configmap.yaml", Data: configmapData}},
Makefile:81: recipe for target 'test-style' failed
make: *** [test-style] Error 1
```

--- a/cmd/helm/upgrade_test.go
+++ b/cmd/helm/upgrade_test.go
@@ -236,7 +236,7 @@ func prepareMockRelease(releaseName string, t *testing.T) (func(n string, v int,
 			Description: "A Helm chart for Kubernetes",
 			Version:     "0.1.0",
 		},
-		Templates: []*chart.File{&chart.File{Name: "templates/configmap.yaml", Data: configmapData}},
+		Templates: []*chart.File{{Name: "templates/configmap.yaml", Data: configmapData}},
 	}
 	chartPath := filepath.Join(tmpChart, cfile.Metadata.Name)
 	if err := chartutil.SaveDir(cfile, tmpChart); err != nil {

--- a/pkg/action/upgrade.go
+++ b/pkg/action/upgrade.go
@@ -40,11 +40,11 @@ type Upgrade struct {
 	ChartPathOptions
 	ValueOptions
 
-	Install   bool
-	Devel     bool
-	Namespace string
-	Timeout   time.Duration
-	Wait      bool
+	Install      bool
+	Devel        bool
+	Namespace    string
+	Timeout      time.Duration
+	Wait         bool
 	DisableHooks bool
 	DryRun       bool
 	Force        bool

